### PR TITLE
Feature/timer

### PR DESCRIPTION
**Pull request by SaldanRoman**
https://b24-lmffg5.bitrix24.ua/workgroups/group/8/tasks/task/view/64/

### CODE REVIEW

File: css/article.css [hunk 1]
```diff
@@ -0,0 +1,85 @@
+nav {
+  box-sizing: border-box;
+}
+
+.nav-links-wrapper {
+  padding: 20px 0;
+}
+.nav-links:hover {
+  transition: 0.3s;
```
Comment (romanivna) [inline css/article.css]: leading zero

File: css/article.css [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+  font-size: 1.07rem;
+  color: #666;
+  display: block;
+  border-bottom: 1px solid #f3f3f3;
+  text-transform: capitalize;
+  padding: 10px 0;
+}
+
+.article-links:hover {
+  transition: 0.3s;
```
Comment (romanivna) [inline css/article.css]: 0

File: css/slider.css [hunk 3]
```diff
@@ -9,7 +9,7 @@
 
 .slider:hover .slider-buttons-wrapper {
   opacity: 1;
-  transition: .3s;
+  transition: 0.3s;
```
Comment (romanivna) [inline css/slider.css]: 0 at whole file

File: css/timer.css [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+  margin: 5px;
+  background-color: #0164c9;
+  color: #fff;
+  font-size: 1.14rem;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 39px;
+  height: 36px;
+  transition: ease-out 0.4s;
```
Comment (romanivna) [inline css/timer.css]: 0

File: js/articles.js [hunk 5]
```diff
@@ -52,7 +52,7 @@ xmlhttp.onreadystatechange = function () {
       articlesDescriptionId.classList.add("articles-description-id");
       articlesDescriptionId.innerHTML = "more details";
       articlesDescriptionId.href =
-        "articles.html?" + data["articles"][articleIndex].id;
+        "article.html?" + data["articles"][articleIndex].id;
```
Comment (romanivna) [inline js/articles.js]: you should find in array element whichhas the same id, not find element by index in array. it's huge error.

File: js/timer.js [hunk 6]
```diff
@@ -0,0 +1,85 @@
...
+      "rotateX(" + rotationClockHours + "deg)";
+    timerDataWrapper[2].style.transform =
+      "rotateX(" + rotationClockMinutes + "deg)";
+    displaySeleTime[2].style.transform =
+      "rotateX(" + rotationClockMinutes + "deg)";
+    rotationClockDays += 180;
+  }
+  displaySeleTime[0].innerText = days;
+
+  // if (displaySeleTime[1].innerText > hours) {
```
Comment (romanivna) [inline js/timer.js]: remove commens

File: js/timer.js [hunk 7]
```diff
@@ -0,0 +1,85 @@
...
+  //     "rotateX(" + rotationClockMinutes + "deg)";
+  //   displaySeleTime[2].style.transform =
+  //     "rotateX(" + rotationClockMinutes + "deg)";
+  //   rotationClockHours += 180;
+  // }
+
+  displaySeleTime[1].innerText = hours;
+
+  // if (displaySeleTime[2].innerText > minutes) {
+  //   timerDataWrapper[2].style.transform =
```
Comment (romanivna) [inline js/timer.js]: comments

File: js/timer.js [hunk 8]
```diff
@@ -0,0 +1,85 @@
...
+  //     "rotateX(" + rotationClockMinutes + "deg)";
+  //   displaySeleTime[2].style.transform =
+  //     "rotateX(" + rotationClockMinutes + "deg)";
+  //   rotationClockMinutes += 180;
+  // }
+
+  displaySeleTime[2].innerText = minutes;
+
+  displaySeleTime[3].innerText = seconds;
+  // if (displaySeleTime[3].innerText == seconds) {
```
Comment (romanivna) [inline js/timer.js]: comments

File: styles.css [hunk 9]
```diff
@@ -4,7 +4,8 @@
 @import "css/product-page.css";
 @import "css/articles.css";
 @import "css/header.css";
-
+@import "css/article.css";
+@import "css/timer.css";
```
Comment (romanivna) [inline styles.css:8]: empty line after imports

**Comment by OlecsandrStelmakh** [comment]
the branch contains files that are not related to it, and which are not in dew:
article.html
css/article.css
js/article.js

and changes in someone else's branch (js/articles.js)

File: css/article.css (hunk 1 above)
Comment (romanivna) [inline css/article.css]: where is it solved? leading zero is deprecated!

File: css/article.css (hunk 2 above)
Comment (romanivna) [inline css/article.css]: same
don't mark as resolved if it's not resolved :) check all other places where's leading zero and remove it

File: js/articles.js [hunk 10]
```diff
@@ -1,63 +1,60 @@
...
-  }
-};
-xmlhttp.open("GET", "jsons/articles.json", true);
-xmlhttp.send();
+  };
+  xhr.open("GET", "./jsons/articles.json", true);
+  xhr.send();
+})();
+
+function bildArticle(articleObject) {
```
Comment (romanivna) [inline js/articles.js]: bUild

File: styles.css (hunk 9 above)
Comment (romanivna) [inline styles.css:8]: where is it solved?
empty line after imports!

**Review by romanivna — COMMENTED** [review]
also discuss with Olexandr his comment and resolve it if it's possible

File: css/slider.css (hunk 3 above)
Comment (romanivna) [inline css/slider.css]: Roman) remove zeroes!) at whole file

**Review by romanivna — COMMENTED** [review]
remove zeroes and if Olexandr is fine - i'm fine

**Review by OlecsandrStelmakh — APPROVED** [review]

**Pull request merged**